### PR TITLE
Refactor dispatching transactions so "Reclaim all" doesn't lose more precision

### PR DESCRIPTION
**Pull request by lukaw3d**

### CODE REVIEW

File: src/app/components/ReclaimEscrowForm/index.tsx [hunk 1]
```diff
@@ -59,8 +60,8 @@ export const ReclaimEscrowForm = memo((props: Props) => {
     dispatch(
       transactionActions.reclaimEscrow({
         type: 'reclaimEscrow',
-        amount: Number(props.maxAmount) / 10 ** 9,
-        shares: Number(props.shares) / 10 ** 9,
+        amount: props.maxAmount,
+        shares: props.shares,
```
Comment (lukaw3d) [inline src/app/components/ReclaimEscrowForm/index.tsx:64]: No rounding :tada:

**Comment by pro-wh** [comment]
- could you add a test for 9,999,999,999,999,999,999 shares
- the APIs these come from, they all round, don't they?

**Comment by lukaw3d** [comment]
Oh, we lose precision when parsing from API..

**Comment by lukaw3d** [comment]
Will revisit in another year.

**Review by pro-wh — APPROVED** [review]
yea

**Pull request merged**